docs/config-description.md

```diff
@@ -1,452 +1,600 @@
 - **Type**: int
-- **Default value**: 512
-- **Constraints**: The minimum value is `0`. The maximum value is `65535`
- -   **Description**: Page fetch size for cache operations.
-### Key: cache.load
+- **Default value**: `512`
+- **Constraints**: The minimum value is `0`. The maximum value is `65535`.
+- **Description**: The page fetch size for cache operations.
+
+### cache.load
+
 - **Required**: True
 - **Type**: string
-- **Default value**: async
-- **Constraints**: The value must be one of the following: `sync, async, none`
- -   **Description**: Cache loading strategy ('sync' or 'async').
-### Key: log_channels.[].channel
+- **Default value**: `async`
+- **Constraints**: The value must be one of the following: `sync`, `async`, `none`.
+- **Description**: The strategy used for Cache loading.
+
+### log_channels.[].channel
+
 - **Required**: False
 - **Type**: string
 - **Default value**: None
-- **Constraints**: The value must be one of the following: `General, WebServer, Backend, RPC, ETL, Subscriptions, Performance, Migration`
- -   **Description**: Name of the log channel.'RPC', 'ETL', and 'Performance'
-### Key: log_channels.[].log_level
+- **Constraints**: The value must be one of the following: `General`, `WebServer`, `Backend`, `RPC`, `ETL`, `Subscriptions`, `Performance`, `Migration`.
+- **Description**: The name of the log channel.
+
+### log_channels.[].log_level
+
 - **Required**: False
 - **Type**: string
 - **Default value**: None
-- **Constraints**: The value must be one of the following: `trace, debug, info, warning, error, fatal, count`
- -   **Description**: Log level for the specific log channel.`warning`, `error`, `fatal`
-### Key: log_level
+- **Constraints**: The value must be one of the following: `trace`, `debug`, `info`, `warning`, `error`, `fatal`, `count`.
+- **Description**: The log level for the specific log channel.
+
+### log_level
+
 - **Required**: True
 - **Type**: string
-- **Default value**: info
-- **Constraints**: The value must be one of the following: `trace, debug, info, warning, error, fatal, count`
- -   **Description**: General logging level of Clio. This level will be applied to all log channels that do not have an explicitly defined logging level.
-### Key: log_format
+- **Default value**: `info`
+- **Constraints**: The value must be one of the following: `trace`, `debug`, `info`, `warning`, `error`, `fatal`, `count`.
+- **Description**: The general logging level of Clio. This level is applied to all log channels that do not have an explicitly defined logging level.
+
+### log_format
+
 - **Required**: True
 - **Type**: string
-- **Default value**: %TimeStamp% (%SourceLocation%) [%ThreadID%] %Channel%:%Severity% %Message%
+- **Default value**: `%TimeStamp% (%SourceLocation%) [%ThreadID%] %Channel%:%Severity% %Message%`
 - **Constraints**: None
- -   **Description**: Format string for log messages.
-### Key: log_to_console
+- **Description**: The format string for log messages.
```

Comment:
I'd suggest to link the docs that describe the format: https://beta.boost.org/doc/libs/1_83_0/libs/log/doc/html/log/tutorial/formatters.html